Options/Applicative.hs

```diff
@@ -14,6 +15,11 @@ module Options.Applicative (
   -- | The standard @Applicative@ module is re-exported here for convenience.
   module Control.Applicative,
 
+  -- | Re-export Data.Monoid's (<>) operator for convenience.
```

Comment:
Hmm. This was entirely to be be consistent with what `optparse-applicative` has been exporting since [0.7](https://hackage.haskell.org/package/optparse-applicative-0.7.0/docs/src/Options-Applicative-Builder.html).

That said, thank you both for the input. I'll probably drop the export completely and let the user decide which instance to use.